helm/loki/values.yaml

```diff
@@ -73,6 +73,8 @@ loki-upstream:
       requests:
         memory: 3072Mi
         cpu: 200m
+    extraArgs:
+      - -querier.multi-tenant-queries-enabled
```

Comment:
This is experimental, are we confident ?
https://github.com/grafana/loki/blob/main/pkg/querier/querier.go#L69

Reply:
oh, I had no idea.

It's well documented: 
* in Grafana 2.6 release notes - https://grafana.com/blog/2022/07/27/grafana-loki-2.6-release/
* in documentation - https://grafana.com/docs/loki/latest/operations/multi-tenancy/
...and Loki people are pushing us to use it.